### DISCUSSION TITLE
How can I explain my transaction ?

### DISCUSSION THREAD

**Discussion by pullmann4rent** [vitaly-t/pg-promise#898] [Announcements]
### Expected behavior

Hello,

if I have an transaction like this:

```
await pg.tx({mode}, t => {
  await t.none('INSERT INTO test ... () Values ()');
  await t.none('INSERT INTO test2 ... () Values ()');
  await t.none('INSERT INTO test3 ... () Values ()');
});
```

How can I explain it now too see how much time it takes ?

### Environment

* Version of pg-promise:
* OS type (Linux/Windows/Mac):
* Version of Node.js:

**Comment by vitaly-t** [comment]
For development, you can use [pg-monitor](https://github.com/vitaly-t/pg-monitor) for this. For production, there's `EXPLAIN` query.

**Comment by vitaly-t** [comment]
@pullmann4rent Did this not answer your question? If it did, then please indicate so.
